pkg/util/k8s/kubeproxy.go

```diff
@@ -255,3 +256,30 @@ func (ka *kpcArgs) getPortRange(key string) string {
 	}
 	return value
 }
+
+// getFeatureGates parses feature-gates and returns a map[string]bool
+func (ka *kpcArgs) getFeatureGates(key string) map[string]bool {
+	val := ka.args[key]
+	if len(val) == 0 {
+		return nil
+	}
+	delete(ka.args, key)
+	featureGates := make(map[string]bool)
+	for _, v := range val {
```

Comment:
you could perhaps do

    featureGates := cliflag.NewMapStringBool(nil)
    for _, v := range val {
            err := featureGates.Set(v)
            if err != nil { ... }
    }
    return featureGates.Map

where `cliflag` is `"k8s.io/component-base/cli/flag"`

Reply:
Thanks. So much more cleaner.